hail/src/main/scala/is/hail/expr/ir/InferPType.scala

```diff
@@ -224,12 +271,6 @@ object InferPType {
           i.pType
         })
         a.implementation.returnPType(pTypes, a.returnType)
-      case a@ApplySpecial(_, _, args, _) =>
```

Comment:
is this unused?

Reply:
Yep. The case above this is `case a: AbstractApplyNode[_] =>` of which `ApplySpecial` is an implementing class.